pkg/Library/NodeGraph/Editor.js

```diff
@@ -87,7 +87,7 @@ toolbarIcons({selectedNodeId}, {graph}) {
 render(inputs, state) {
   return {
     graph: this.renderGraph(inputs, state),
-    graphRects: state.layout || this.simpleLayout(state)
+    graphRects: state.layout //|| this.simpleLayout(state)
```

Comment:
i'll remove the `simpleLayout` method too.. 
the default positions (and the size!) are now assigned by the node-graph, right? just doublechecking i read it correctly.

Reply:
Yes, there was an old position generator in node-graph that I rehydrated, since it already had parameters. Yes, then I added size forcing based on i/o count.